### ISSUE TITLE
Windows Hello in Bitwarden Desktop Application version 1.33 not functioning properly

### ISSUE THREAD

**Issue by TEngelman** [bitwarden/clients#2679]
### Steps To Reproduce

Please take a look at my report on:
https://community.bitwarden.com/t/windows-hello-in-bitwarden-desktop-application-version-1-33-not-functioning-properly/40806

### Expected Result

The Edge browser add-on settings should be able to set on "unlock with biometrics".

### Actual Result

When setting the Edge browser add-on to "unlock with biometrics", the response is "Browser intergration is not enabled".
So I can no longer use biometrics unlock in Edge.

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/102973243/168092822-be821e96-e1ec-4faa-889d-c184664ed156.png)
![image](https://user-images.githubusercontent.com/102973243/168093903-c28c4686-b3a2-418e-9d4b-22622474d2f5.png)

![image](https://user-images.githubusercontent.com/102973243/168093109-f6b35c97-d138-412f-bf27-2260ff3908eb.png)

### Additional Context

The Firefox browser add-on works fine.

### Operating System

Windows

### Operating System Version

|Edition|Windows 11 Education| |Version|21H2| |Installed on|‎06/‎04/‎2022| |OS build|22000.652| |Experience|Windows Feature Experience Pack 1000.22000.652.0|

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 1.33.0 Shell 16.1.0 Renderer 96.0.4664.174 Node 16.9.1 Architecture x64

**Comment by TEngelman** [comment]
Hi there,
It has been 28 days now since I opened this issue and up t now I haven't seen any action or progress from anyone. What's going on?

Meanwhile, I just updated the desktop version to:
Version 2022.5.1
Shell 16.1.0
Renderer 96.0.4664.174
Node 16.9.1
Architecture x64
The browser extension for MS Edge:  
Bitwarden
© Bitwarden Inc. 2015-2022
Version: 1.58.0
still doesn't accept the setting "Unlock with biometrics",  and reports: 
![image](https://user-images.githubusercontent.com/102973243/172852082-40534f01-65b1-4303-85fa-7ec50a089e33.png)

The browser extension for Mozilla Firefox:
Bitwarden
© Bitwarden Inc. 2015-2022
Version: 1.58.0
works properly as expected!

When will you respond to this issue en solve it?

Regards,
Theo

**Comment by richard378** [comment]
I have this same problem.  It is fussy and sometimes works and sometimes doesn't.  Currently I can't get it to work in any browser.  Edge, Firefox, Comodo Dragon, etc.  Sometimes I can get it working in one or more browsers but it is not consistent.

**Comment by Hinton** [comment]
Have you tried turning the feature on and off again which should re-generate the manifest files.

**Comment by richard378** [comment]
I have tried disabling and enabling it many times including uninstalling and reinstalling everything including browser extensions.

**Comment by kortov** [comment]
related to #2435

**Comment by TEngelman** [comment]
Today, finally my issue has been solved, thanks to Sugianto from the Bitwarden support staff.
Here's the scenario:

1. Open Edge
2. Right-click the icon, then "Manage Extension"
3. Enable "Allow access to file URLs"
4. Start the extension and authenticate
5. Close the browser and start it again
6. Open the desktop app settings
7. De-select the "Browser Integration" setting
8. Close the desktop app settings to be sure to save the new setting
9. Open the desktop app settings again
10. Select the "Browser Integration" setting
11. Close the desktop app settings again to be sure to save the new setting
12. By doing so and then enabling Biometrics unlock in the extension the response was two-fold;
-	The extension showed the “Browser integration not enabled in the desktop app” error
-	The Windows system Biometric authentication popped up and I used my fingerprint sensor.
Immediately the setting in the extension changed to
![image](https://user-images.githubusercontent.com/102973243/192100290-0c9c102d-f0c1-45bf-ae49-34e5b7e07352.png)

To be sure I tested the extension by closing the browser and starting it again.
Then I opened the extension which immediately prompted me for Biometric authentication, which worked OK.
As a last action, I unchecked the “Allow access to file URLs” setting in the "Manage Extension" settings, closed the browser and restarted it.
When opening the extension, the Biometric authentication worked as intended.

Sugianto explained to me that Bitwarden staff has not been able to reproduce this issue #2679, nor the similar issues #2435 and #2942.
That's why I was asked to perform several trial-and-error actions before the upper scenario turned out to be successful.

I will close this issue now.
Theo

**Comment by CarstenG2** [comment]
Did not work here. Same error as usual "Browser integration is not set up".

**Comment by AizatHawari** [comment]
Not working too

**Comment by kesavan-ak** [comment]
One minor thing missed, after opening Manage Extensions, Navigate to Bitwarden **Details**, in that section you should click allow access to file URLs. In edge, the main 'Managing Extension' Page also has an Allow URL option. Don't confuse that with this.
![Screenshot_30-4-2024_12246_extensions](https://github.com/bitwarden/clients/assets/168517612/777a8b73-6bf7-4057-acd8-e78b217b1509)
![Screenshot_30-4-2024_122512_extensions](https://github.com/bitwarden/clients/assets/168517612/c5532c0b-c8a5-4ec7-8ab8-c1e2e363bd5d)

**Comment by zim29** [comment]
Hello everyone,

I hope this helps others and provides developers with insights into the issue.

As of today, January 22, 2025, I downloaded the Microsoft Store version of Bitwarden. However, the browser integration failed to work. I tried several solutions suggested in this thread, but unfortunately, none of them resolved the problem.

To fix the issue, I uninstalled the Bitwarden Desktop App and downloaded the Standard Installer directly from the Bitwarden website. This successfully resolved my problem.

I hope this information is helpful!
